packages/@sfx/sayt-driver-plugin/test/unit/common/sayt-driver-plugin.test.ts

```diff
@@ -202,11 +202,18 @@ describe('Sayt Driver Plugin', () => {
     });
 
     it('should make a search call through the sayt client', () => {
-      driver.sendSaytApiRequest(saytDataPayload);
+      const payload = {
```

Comment:
Should probably change the original variable setup in the `beforeEach()` step instead and then modify the two tests accordingly.